study.md

```diff
@@ -44,21 +50,23 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+view means the HTML fragents that makes up users see has the screen or page of the site.
+view state means the view that the users, in spa everything what the user see is "view"
+Router allow the user to go from View A to View B within the browser.
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+It would feel more app like and ease of response for the user.
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+JavaScript, JSON, and AJAX
```

Comment:
Front end frameworks are also a super useful tool when developing a SPA. Things like React, Angular, and Ember.